server/odc-service/src/main/java/com/oceanbase/odc/service/flow/task/PreCheckRuntimeFlowableTask.java

```diff
@@ -196,6 +196,7 @@ protected void onFailure(Long taskId, TaskService taskService) {
         } catch (Exception e) {
             log.warn("Failed to store task result", e);
         }
+        super.onFailure(taskId, taskService);
```

Comment:
What's the consideration for adding this line and removing `super.onSuccessful(taskId, taskService);`?